code-quality/accessibility/README.md

```diff
@@ -0,0 +1,74 @@
+# Accessibility
+
+## What is it?
+
+Accessibility is basically making sure web sites care fully inclusive and can be properly access for **everyone**, not just people with disabilities.
+
+The main standard for web accessibility is the Web Content Accessibility Guidelines (WCAG). The main standard is [WCAG 2.0](https://www.w3.org/TR/WCAG20/), but [WCAG 2.1](https://www.w3.org/TR/WCAG21/) came out in June 2018 and will be the main standard going forward.
+
+## Why?
+
+Various governments have accessibility requirements going forwards. Some countries, like Australia have a compliance requirement for all content. Europe has updates its directive requiring compliance on all public sector (non-government) site or risking fines or legal penalties. In the USA, there are accessibility requirements for Information and Communications Technology (ICT) that was updated in January of 2017 in [this article about section 508 of the Rehibilitiation Act](https://www.access-board.gov/guidelines-and-standards/communications-and-it/about-the-ict-refresh/overview-of-the-final-rule). This only applies to federal, state, and local government sites. Public sector sites in the US are covered by Title III of the Americans with Disabilities Act. This title has not been updated to include websites.... *yet*.
+
+While this may mean that there is no standard for non-government sites, that does not prevent someone from filing a lawsuit because your site is not able to be used properly by someone with a disability.
+
+## WCAG 2.1 Summary
+
+I would recommend giving the full standard a read, but here is a main summary of some of the main questions your site would have to answer. The criteria falls into four critera: Perceivable, Operable, Understandable and Robust. There are three standards, A, AA, and AAA. Most compliance requirements ask for level AA compliance.
```

Comment:
`Shift3 recommends giving the full standard a read. The criteria to meet the standard falls into four categories: **Perceivable**, **Operable**, **Understandable** and **Robust**. There are three ratings of compliance, A, AA, and AAA. Most compliance requirements ask for level AA compliance.`